### PR TITLE
Completed login UI

### PR DESCRIPTION
**Pull request by Twdckcv**
This PR has the CSS, Login.tsx and images

### CODE REVIEW

**Comment by The-TT-Hacker** [comment]
css, fonts and images should be in the public directory not the src directory

File: frontend/src/pages/Login.tsx [hunk 1]
```diff
@@ -1,10 +1,35 @@
...
 
 const Login = () => {
   return (
-    <div>
-      <p>Login with Github</p>
-    </div>
+    <html>
+      <body className="login-background">
+        <div id="login-page" className="row">
+          <div className="col s12 z-depth-4 card-panel">
```
Comment (The-TT-Hacker) [inline frontend/src/pages/Login.tsx]: use react-bootstrap which gives you Col and Row components instead of using classes

**Comment by The-TT-Hacker** [comment]
yeah look youve added way to much stuff for 1 merge request. remove all the styling, jquery, fonts stuff to separate MRs so they can be reviewed on their own. I just want to see the functional login UI on its own

**Comment by dn1470** [comment]
Should we just close this request and make another pr? Like just adding css?

**Comment by The-TT-Hacker** [comment]
you can just git reset back to main and only commit the login components and anything it really needs

File: frontend/src/css/style.css [hunk 2]
```diff
@@ -0,0 +1,425 @@
...
+  font-size: 1.7rem;
+  font-weight: 500;
+  margin-top: 5px;
+  margin-bottom: 0;
+}
+
+/*----------------------------------------
+    Login Page
+------------------------------------------*/
+.login-form {
```
Comment (The-TT-Hacker) [inline frontend/src/css/style.css]: this isnt how you do css in component based frameworks. make a css file in the same directory as the components using css modules https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/

Comment (The-TT-Hacker) [reply frontend/src/css/style.css]: im really against huge amounts of CSS as well. just keep it simple for this

Comment (The-TT-Hacker) [reply frontend/src/css/style.css]: theres no way you need all this crap for a simple login page

**Comment by The-TT-Hacker** [comment]
Can you move the materialize, and fonts stuff to another PR

File: frontend/src/pages/Login.tsx [hunk 3]
```diff
@@ -1,10 +1,42 @@
...
 const Login = () => {
   return (
-    <div>
-      <p>Login with Github</p>
-    </div>
+    <html>
+      <body className="login-background">
+        <div id="login-page">
+          <div className="col s12 z-depth-4 card-panel">
+            <div className="row">
```
Comment (The-TT-Hacker) [inline frontend/src/pages/Login.tsx]: use this https://react-bootstrap.github.io/layout/grid/

File: frontend/src/pages/Login.tsx [hunk 4]
```diff
@@ -1,10 +1,42 @@
...
+                <h4 className="login-form-text">Use your Github Account</h4>
+              </div>
+            </div>
+            <div className="row">
+              <div className="login-box">
+                <a
+                  href="http://localhost:3000/login"
+                  className="btn-large waves-effect waves-teal col s12"
+                >
+                  <span />
```
Comment (The-TT-Hacker) [inline frontend/src/pages/Login.tsx]: use css instead of having multiple spans

File: frontend/tsconfig.json [hunk 5]
```diff
@@ -22,6 +22,7 @@
     "jsx": "react-jsx"
   },
   "include": [
+    "src/login",
```
Comment (The-TT-Hacker) [inline frontend/tsconfig.json]: you dont need this

File: frontend/src/pages/login/Login.tsx [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+              </div>
+            </div>
+            <div className="row">
+              <div className={styles.login_box}>
+                <a
+                  href="http://localhost:3000/login"
+                  className="btn-large waves-effect waves-teal col s12 black"
+                >
+                  <span></span>
+                  <span></span>
```
Comment (The-TT-Hacker) [inline frontend/src/pages/login/Login.tsx]: use css instead of all these spans

File: frontend/public/index.html [hunk 7]
```diff
@@ -23,10 +20,11 @@
...
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Go Lambda Skeleton</title>
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-  </body>
-</html>
+  <title>Very Secured</title>
```
Comment (The-TT-Hacker) [inline frontend/public/index.html]: Title should Dyno? seems like thats what you guys are calling it?

File: frontend/package.json [hunk 8]
```diff
@@ -45,6 +46,7 @@
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^14.2.6",
     "@types/jest": "^28.1.6",
+    "@types/materialize-css": "^1.0.11",
```
Comment (The-TT-Hacker) [inline frontend/package.json:47]: im hesitant to include both bootstrap and meterialize. 2 CSS frameworks, choose one of the other

File: frontend/src/pages/login/page-center_login.module.css [hunk 9]
```diff
@@ -0,0 +1,14 @@
+html,
+body {
+  height: 100%;
+}
+
+html {
```
Comment (The-TT-Hacker) [inline frontend/src/pages/login/page-center_login.module.css]: should be doing stuff with html and body in a compontents css. do it on divs

Comment (dn1470) [reply frontend/src/pages/login/page-center_login.module.css]: does it mean we should move these lines to login.module.css?

**Pull request merged**